tools/generator/AnDOGenerator.py

```diff
@@ -123,14 +123,14 @@ def __init__(self, expName=None, guid=None, sesID=None, date=None, sesNumber=Non
     def get_session_path(self):
         path = os.path.join(f'exp-{self.expName}',
                             f'sub-{self.guid}',
-                            f'ses-{self.sesID}')
+                            f'ses-{self.sesID}',
+                            f'ephys')
         return path
 
     def get_all_folder_paths(self):
         paths = []
         session = self.get_session_path()
-        for datafolder in ['rawdata', 'metadata', 'derivatives']:
-            paths.append(os.path.join(session, datafolder))
+        paths.append(session)
```

Comment:
keep the datafolder list here for now.

Reply:
Updated now datafolder list just contain "ephys"